doc/src/stdlib/file.md

```diff
@@ -50,6 +50,7 @@ Base.Filesystem.basename
 Base.@__FILE__
 Base.@__DIR__
 Base.@__LINE__
+Base.propagate_sourceloc!
```

Comment:
I've moved these around - hopefully into a more coherent location in the stdlib docs.